cfgov/v1/jinja2/v1/layouts/layout-1-3.html

```diff
@@ -20,12 +16,7 @@
         <div class="u-layout-grid_wrapper">
             {% block hero -%}{%- endblock %}
             {% block pre_content scoped -%}
-                {% if breadcrumb_items | length > 0 %}
-                    <div class="u-layout-grid_breadcrumbs">
-                        {%- import 'v1/includes/molecules/breadcrumbs.html' as breadcrumbs with context -%}
-                        {{ breadcrumbs.render(breadcrumb_items) }}
-                    </div>
-                {% endif %}
+                {% include '_breadcrumbs.html' %}
```

Comment:
Could I please request that we use the "absolute" path to this (`'v1/layouts/_breadcrumbs.html'`) instead of the relative one (here, and in the other instance below)? See comment [here](https://github.com/cfpb/consumerfinance.gov/blob/b8bf9867f7f8feef45987bcb2f5e1d72894ca3be/cfgov/v1/jinja2_environment.py#L15-L39) for rationale -- loading via relative paths is something bespoke to cf.gov which ideally we could deprecate someday.

Reply:
Ah yes, that also makes it consistent with the full layout that was already using the absolute path. Done in https://github.com/cfpb/consumerfinance.gov/pull/8233/commits/eb583e012ba2ab4e1b14b5d2599706b016d1f37b